src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable global-require */
+// NOTE: disable global-require for dummy data, will be removed
+// once data is fed top-down
+
+import React from "react";
+import FilterBar from "../FilterBar/FilterBar";
+import { MovieProps } from "../../components/Movie/Movie";
+import MovieList from "../MovieList/MovieList";
+
+import styles from "./Dashboard.module.scss";
+
+// DATA
+
+const genres = ["All", "Documentary", "Comedy", "Horror", "Crime"];
+const sortOptions = ["Release date", "Rating", "Runtime", "Title"];
+
+const movies: MovieProps[] = [
```

Comment:
Under "src/core" folder that you create after one of the above comments consider creating a folder "data" where you would store such arrays/objects to use across the app.